src/compute/src/render/top_k.rs

```diff
@@ -439,8 +439,8 @@ where
         };
         // Turn input into collection.
         let input = input.as_collection(|k, v| (k.into_owned(), v.into_owned()));
-        // Negate oks and concatenate them with the input.
-        (oks.negate().concat(&input), errs)
+        // Concatenate oks with the input.
```

Comment:
Nit and not on you, so feel free to ignore, but this comment just re-states the next line of very obvious code without adding any additional insight, so it seems unnecessary. Same for the "Turn input into collection." one above.

Reply:
Agreed. Removing both comments.